src/renderer/views/wallet/AssetsView.tsx

```diff
@@ -16,7 +18,10 @@ export const AssetsView: React.FC = (): JSX.Element => {
   const history = useHistory()
   const { chainBalances$ } = useWalletContext()
 
-  const chainBalances = useObservableState(chainBalances$, [] as ChainBalances)
+  const chainBalances = FP.pipe(
+    useObservableState(chainBalances$, [] as ChainBalances),
+    A.filter((chainBalance) => RD.isSuccess(chainBalance.balances))
+  )
```

Comment:
^ That's not correct if we still want to show loading and error states of each chain.